pkg/envoy/cds/builder.go

```diff
@@ -204,6 +329,31 @@ func getPrometheusCluster() *xds_cluster.Cluster {
 	}
 }
 
+func getTracingCluster(address string, port uint32) *xds_cluster.Cluster {
+	return &xds_cluster.Cluster{
+		Name:        constants.EnvoyTracingCluster,
+		AltStatName: constants.EnvoyTracingCluster,
+		ClusterDiscoveryType: &xds_cluster.Cluster_Type{
+			Type: xds_cluster.Cluster_LOGICAL_DNS,
+		},
+		LbPolicy: xds_cluster.Cluster_ROUND_ROBIN,
+		LoadAssignment: &xds_endpoint.ClusterLoadAssignment{
+			ClusterName: constants.EnvoyTracingCluster,
+			Endpoints: []*xds_endpoint.LocalityLbEndpoints{
+				{
+					LbEndpoints: []*xds_endpoint.LbEndpoint{{
+						HostIdentifier: &xds_endpoint.LbEndpoint_Endpoint{
+							Endpoint: &xds_endpoint.Endpoint{
+								Address: envoy.GetAddress(address, port),
```

Comment:
super nit: I'd consider just setting the address on the builder as the value of envoy.GetAddress